src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/DeferredProjectsUtility/DeferredProjectVSUtility.cs

```diff
@@ -41,7 +41,7 @@ public static bool IsSolutionDPLEnabled()
         {
 #if VS14
             // for Dev14 always return false since DPL not exists there.
-            return Task.FromResult(false);
+            return false;
```

Comment:
do we need to do this at [here](https://github.com/NuGet/NuGet.Client/blob/c323f55e1874ab7caa0f1add371ab422560347f1/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/DeferredProjectsUtility/DeferredProjectVSUtility.cs#L24) and [here](https://github.com/NuGet/NuGet.Client/blob/c323f55e1874ab7caa0f1add371ab422560347f1/src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/DeferredProjectsUtility/DeferredProjectVSUtility.cs#L66)  ??